Framework/MDAlgorithms/src/IntegratePeaksMD2.cpp

```diff
@@ -57,9 +57,6 @@ void IntegratePeaksMD2::init() {
                                                            Direction::Input),
                   "An input MDEventWorkspace.");
 
-  std::vector<std::string> propOptions{"Q (lab frame)", "Q (sample frame)",
-                                       "HKL"};
-
   declareProperty(
```

Comment:
This variant of the algorithm uses the coordinate system already defined on the workspace so it doesn't need those options. I think it's a copy-and-paste error from version 1.